it/router/RouterSaUnresponsiveFeature.scala

```diff
@@ -43,17 +42,21 @@ class RouterSaUnresponsiveFeature extends StubbedFeatureSpec with CommonStubs {
       Then("the user should be routed to BTA Home Page")
       on(BtaHomePage)
 
-      And("the authority object should be fetched once for AuthenticatedBy")
-      verifyAuthorityObjectIsFetched()
+      eventually {
```

Comment:
i would have thought that fetching the auth object, and the following calls inside this `eventually` block occur before the user eventually arrives at the `BtaHomePage`. If so, that suggests that the `eventually {` line should be moved upwards by a few lines